modern-business/inc/fse-template-data.php

```diff
@@ -29,41 +87,49 @@ public function insert_default_template_data() {
 			return;
 		}
 
+		// Set header and footer content based on data fetched from the WP.com API.
+		$this->fetch_template_parts();
+
+		// Avoid creating template parts if data hasn't been fetched properly.
+		if ( empty( $this->header_content ) || empty( $this->footer_content ) ) {
```

Comment:
Are we disregarding the possibility of a site without some template parts (e.g. no footer)?
What about an `&&` here?

Reply:
I was thinking about the same thing while writing this code, and I don't have a strong preference yet. I went for `||` at the time because all of the themes from our current scope provide both, and if that's not the case something has gone wrong in the process and we better bail.

In general case though `&&` might make more sense because of added flexibility. On the other hand, having an FSE editor that doesn't show the footer goes against our goal of matching the content in the editor and the front end as closely as possible? 🤔 I'm thinking we could make it a minimum requirement for themes to provide a header and footer.

Reply:
I think the current logic is 👍 We can revisit if we ever end up with a theme that doesn't have a footer.